misc/ballerinalang-data-mapper/src/main/java/org/ballerinalang/datamapper/AIDataMapperCodeActionUtil.java

```diff
@@ -127,21 +115,22 @@ public static AIDataMapperCodeActionUtil getInstance() {
      * @return edits for the data mapper code action
      * @throws IOException throws if error occurred when getting generatedRecordMappingFunction
      */
-    public List<TextEdit> getAIDataMapperCodeActionEdits(DiagBasedPositionDetails positionDetails,
-                                                         CodeActionContext context, Diagnostic diagnostic)
+    public ArrayList getAIDataMapperCodeActionEdits(DiagBasedPositionDetails positionDetails,
+                                                  CodeActionContext context, Diagnostic diagnostic)
             throws IOException {
-        List<TextEdit> fEdits = new ArrayList<>();
+        ArrayList outputArray = new ArrayList();
+        JsonArray outputJson = new JsonArray();
```

Comment:
Why do we need this `outputJson`? Is it redundant?

Reply:
Removed the `Output Json`